Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -907,7 +962,13 @@ define([
 
             ++tileProvider._usedDrawCommands;
 
-            command.debugShowBoundingVolume = (tile === tileProvider._debug.boundingSphereTile);
+            if (tile === tileProvider._debug.boundingSphereTile) {
+                if (defined(surfaceTile.orientedBoundingBox)) {
+                    createDebugOrientedBoundingBox(surfaceTile.orientedBoundingBox, Color.RED).update(context, frameState, commandList);
```

Comment:
There is only one reference to the debug primitive.  Don't we need a reference per tile?